ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
+  openstack-installer:
+    name: '4.12'
+    namespace: ocp
+    tag: openstack-installer
+  openstack-tests:
+    name: '4.13'
+    namespace: ocp
+    tag: openstack-tests
+releases:
+  initial:
+    prerelease:
+      product: ocp
+      version_bounds:
+        lower: 4.12.0-0
+        upper: 4.13.0-0
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: '4.13'
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: e2e-openstack-sdn-upgrade
```

Comment:
looks like this one will test with ovn which is the default , shoud the sdn me removed from the name?
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/upgrade/openstack/loki/openshift-upgrade-openstack-loki-workflow.yaml

Reply:
You're right, I'll do it.